peloton-to-garmin.py

```diff
@@ -96,4 +108,3 @@
 
 logger.info("Done!")
 logger.info("Your Garmin TCX files can be found in the Output directory: " + output_directory)
-os.system("pause")
```

Comment:
I had to remove this to ensure the container would run and exit automatically

Reply:
Let's make this configurable. Non-docker users may prefer to still have the output window to see any errors that were written to the console.  Maybe we can put it behind a `DebugMode` flag in the config.ini?  Default could be `false`, i.e. don't pause the console.

Reply:
Alright, I added a flag and also changed the pause command to something that works on a mac and hopefully is platform agnostic.